lib/api/docs.rb

```diff
@@ -1,3 +1,4 @@
 module Api
-  Docs = ::OpenApi::Docs.new(Dir.glob(Rails.root.join("public/doc/swagger*.yaml")))
+  Docs = ::OpenApi::Docs.new(Dir.glob(Rails.root.join("public/doc/swagger*.yaml")) +
+                             Dir.glob(Rails.root.join("public/doc/openapi*.yaml")))
```

Comment:
In catalog, and in the IPP, I believe we want to go to .json format for the new openapi file.

Reply:
yep, cleaner too. moved to openapi*.json for the OpenAPI 3.x specifications.